SDK/AppCenterData/Microsoft.AppCenter.Data.Android/ConversionExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using Com.Microsoft.Appcenter.Data.Models;
+using Newtonsoft.Json;
+
+namespace Microsoft.AppCenter.Data
+{
+    public static class ConversionExtensions
+    {
+        private static readonly DateTime UnixEpoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
+
+        public static DocumentWrapper<T> ToDocumentWrapper<T>(this AndroidDocumentWrapper documentWrapper)
+        {
+            if (documentWrapper.Error != null)
+            {
+                throw new DataException(documentWrapper.Error.Message, documentWrapper.Error);
+            }
+            var deserializedValue = JsonConvert.DeserializeObject<T>(documentWrapper.JsonValue);
```

Comment:
Please check if there is a built-in cast like with NSDate -> DateTime in iOS

Reply:
It's just `long` on Android